CHANGES.txt

```diff
@@ -3,12 +3,14 @@ V3.6.1
 user:
   - Streaming protocols based on new streaming (generation steps) are resumable
   - pyworkflowtest fixed
+  - Avoiding a double process when launching a workflow using the protocol form
```

Comment:
May be you can comment on the new param "executable" in runJob method?

Reply:
I don't get the point/need of this new parameters to runJob (and the rest of related functions)

Reply:
This has been triggerred by the fact that warp has a "bash" script that is called when activating the environment.

We always call sh, and the way to call bash is to either:

make the command bash + arguments or offer this option.

Warp activation environment does not work on sh shells.

Reply:
Ok, in my opinion, it would be easier to just call bash as the first argument without introducing a new parameter.

Reply:
I know but the code is clearer like WarpTools as a program instead of being the program bash.

Still, developers can do both now.